core/templates/dev/head/pages/admin/activities_tab/AdminDevModeActivitiesTabDirective.js

```diff
@@ -104,6 +104,24 @@ oppia.directive('adminDevModeActivitiesTab', [
           }
         };
 
+        $scope.generateDummyExplorations = function() {
+          // Generate dummy explorations with random title.
+          AdminTaskManagerService.startTask();
+          $scope.setStatusMessage('Processing...');
```

Comment:
Do you want to validate that the publish count is less than or equal to the total count?

It might also not be entirely clear that the "exps to publish" are part of the "exps to generate" (as opposed to an addition to those). Another way to frame it, so that you don't have to do validation, is to say "generate N public exps and M private exps".